docs/pages/machine-id/getting-started.mdx

```diff
@@ -0,0 +1,244 @@
+what roles exist on your system.
+</Details>
+<Details scope={["oss","enterprise"]} scopeOnly={true}>
+Connect to the Teleport Auth Server and use `tctl` to examine what roles exist on
+your system.
+</Details>
+
+```code
+$ tctl get roles --format=text
+```
+
+You will see something like the output below on a fresh install of Teleport with the
+default roles—your cluster may have different roles. In this example, let's
+assume you want to give the bot the `access` role to allow it to connect to
+machines within your cluster.
+
+```
+Role    Allowed to login as                           Node Labels Access to resources
+------- --------------------------------------------- ----------- ----------------------------------------
+access  {{internal.logins}}                           <all nodes> event:list,read,session:read,list                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
+auditor no-login-6566121f-b602-47f1-a118-c9c618ee5aec             session:list,read,event:list,read                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
+editor                                                            user:list,create,read,update,delete,...
+```
+
+Machine ID can join with a token or the [IAM Method](https://goteleport.com/docs/setup/guides/joining-nodes-aws) on AWS.
+
+<Tabs>
+  <TabItem label="Token-based Joining">
+  ```code
+  $ tctl bots add robot --roles=access
+  ```
+  </TabItem>
+  <TabItem label="IAM Method">
+  First, create an IAM method token that specifies the AWS account from which
+  the bot can join.
+
+  ```
+  kind: token
+  version: v2
+  metadata:
+    # The token name is not a secret because instances must prove that they are
+    # running in your AWS account to use this token.
+    name: iam-token
+    # Set a long expiry time. The default for tokens is only 30 minutes.
+    expires: "3000-01-01T00:00:00Z"
```

Comment:
Should we make this 1 - 2 yrs? vs 292 years?

Reply:
There's no reason for these to be short-lived since they're not used as a security control at all, regular IAM/EC2 joining work identically. I wonder if it's possible to unset `expires` entirely?

Reply:
The automatically-created token from `tctl bots add ...` for regular (renewable, non-IAM) use is only valid for 1 hour, which is hopefully sane.

Reply:
I think the language here is a bit confusing. This is how long the IAM method joining token is valid for which can be valid for a long time without security concerns.